Sources/SpelledPitch/PitchSpeller/Wetherfield/Wetherfield.swift

```diff
@@ -14,6 +15,98 @@ protocol PitchSpellingNode: Hashable {
     var index: Index { get }
 }
 
+struct Reducer<S, A>: Monoid {
+    static var identity: Reducer {
+        return .init { _, _ in return }
+    }
+    
+    static func <> (lhs: Reducer, rhs: Reducer) -> Reducer {
+        return Reducer { s, a in
+            rhs.reduce(&s, a)
+            lhs.reduce(&s, a)
+        }
+    }
+    
+    let reduce: (inout S, A) -> ()
+}
+
+private enum GraphMask {
+    case basis(WeightedGraph<Pitch.Class, Double>)
+    case adjacency(Graph<Int>)
+    case scaling(WeightedGraph<Int, Double>)
+}
+
+struct Getter<A, B> {
```

Comment:
In some libraries, I have seen this labeled:

```Swift
struct Getter <Whole,Part> { ... }
```

At least for me, this makes things a little more digestible. My intuition is that the digestibility also composes and pays off down the line, if say you have:

```Swift
extension Getter where Whole == W, Part == P { ... }
```

Rather than:

```Swift
extension Getter where A == NotSure, B == WhichIsWhich { ... }
```